src/utils/translations.js

```diff
@@ -0,0 +1,31 @@
+
+import i18n from '@/i18n'
+
+// variables are named according to the general concept of ie (item = sheet, item);  (manifest = document, manuscript)
+const navButtonsDefaultTextArray = ['next_item', 'previous_item', 'next_manifest', 'previous_manifest']
+
+export function areNavButtonsLabelsInConfig(config) {
```

Comment:
I think it should be fine fow now. What we want is actually a generic function `getTranslation` which looks up a translation string and tries to find it either in the user config or in the i18n files. Basically the `t` function is already something like that. It just doesn't check the user config. So maybe there is a way to just extend `t` or (worse) wrap `t` in a custom function and call it `$$t` or so.